machine.go

```diff
@@ -207,7 +197,7 @@ func (cfg *Config) Validate() error {
 		Int64Value(cfg.MachineCfg.MemSizeMib) < 1 {
 		return fmt.Errorf("machine needs a nonzero amount of memory")
 	}
-	if cfg.MachineCfg.HtEnabled == nil {
+	if cfg.MachineCfg.Smt == nil {
 		return fmt.Errorf("machine needs a setting for ht_enabled")
 	}
```

Comment:
- The error has to be updated as well
- Doesn't it have a default value?

Reply:
true. i'll see if it's safe to remove this check